Marker/Marker/Parser/String+Extensions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  String+Extensions.swift
+//  Marker
+//
+//  Created by Htin Linn on 10/27/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Shorthand for `String.Index` used in parsing.
+internal typealias Index = String.Index
+
+internal extension String {
```

Comment:
minor, but in some places you define extensions as explicitly internal, whereas in others you leave the protection level ommitted.

Reply:
Yeah. I'll update this throughout the app to be consistent.